include/NovelRT.h

```diff
@@ -0,0 +1,116 @@
+#ifndef NOVELRT_H
+#define NOVELRT_H
+
+
+typedef void (*NovelUpdateSubscriber)(double);
+namespace NovelRT::Maths
+{
+  template<typename T>
+  class GeoVector;
```

Comment:
We could probably remove the typedefs for the "value types", since they shouldn't be passed around as pointers anyways (and so the typedefs don't matter)